backend/src/api/v1/users/routes.py

```diff
@@ -1,11 +1,12 @@
 from fastapi import APIRouter
 
-from src.api.deps import CurrentUser
-from src.api.v1.users.schemas import UserMe
+from src.api.deps import Session
+from src.api.v1.users.schemas import UserEmail
+from src.api.v1.users.service import is_email_registered
 
 router = APIRouter(prefix="/users", tags=["Users"])
 
 
-@router.get("/me", response_model=UserMe)
-def me(current_user: CurrentUser):
-    return current_user
+@router.get("/exists")
```

Comment:
is this function needed in the api?

Reply:
when resetting user password, you first need to enter an email in the UI, it would be nice to check if such email exists before

Reply:
Yes, I understand that. but is this endpoint needed specifically in the api?

Reply:
yes